lib/ContextualPopup/ContextualPopup.js

```diff
@@ -532,18 +545,32 @@ module.exports = kind(
 	/**
 	* @private
 	*/
+	updateScrim: function (old, value, source) {
+		// We sync modal and spotlightModal here because binding doesn't guarantee sequence
+		// when it is used with observers.
+		if (source == 'modal') this.set('spotlightModal', this.modal);
```

Comment:
Seems like this will trigger `updateScrim` multiple times as we are setting the value of another of its observed properties - would like to avoid that if possible.